SECURITY.md

```diff
@@ -0,0 +1,35 @@
+# Security Policy
+
+The VA.gov team takes security issues in VA.gov seriously. We appreciate your efforts to responsibly 
+disclosure your findings, and will make every effort to acknowledge your contributions.
+
+## Supported Versions
+
+We release patches for security vulnerabilities. Which versions are eligible receiving such patches depend on the 
+[CVSS v3.0](https://www.first.org/cvss/calculator/3.0) Rating:
+
+| Version | Supported          |
```

Comment:
should this be "Score" rather than "Version"?

and "Eligible" rather than "Supported"?